ui/pages/confirm-encryption-public-key/confirm-encryption-public-key.container.js

```diff
@@ -43,9 +40,7 @@ function mapStateToProps(state) {
     fromAccount,
     requester: null,
     requesterAddress: null,
-    conversionRate: useNativeCurrencyAsPrimaryCurrency
-      ? null
-      : conversionRateSelector(state),
+    conversionRate: null,
```

Comment:
Can you help me understand why conversion rate should always be null here?

Reply:
This UI will show by default the crypto amount instead of checking the primary currency setting to decide whether to show fiat or crypto; instead of putting null, the prop should have been totally removed, will update this;
Adding screenshot for reference as how this UI will look like
![image](https://github.com/user-attachments/assets/7848099f-282a-4ed5-9c9c-81df9aeaf11b)